appimage/AppImageBuilder.yml

```
@@ -48,7 +48,12 @@ AppDir:
         key_url: http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x871920D1991BC93C
 
     include:
-      - gir1.2-appindicator3-0.1
+      - gir1.2-ayatanaappindicator3-0.1
+      - libayatana-appindicator3-1
+      - libdbusmenu-glib4
+      - libdbusmenu-gtk3-4
+      - libindicator3-7
+      - libappindicator3-1
```

This is not incorrect, but I think it's best that we only include `gir1.2-ayatanaappindicator3-0.1` here, and then let AppImageBuilder automatically pick up the other dependencies. That way, in the unlikely scenario the list of dependencies change, we don't have take care of them manually.

```suggestion
      - gir1.2-ayatanaappindicator3-0.1
```